stats_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestStatsCopy(t *testing.T) {
+	s := Stats{
+		ServerStats{
+			1,
+			Redirects{
+				2,
+				3,
+			},
+			[]Handler{Handler{"qwe", 4}, Handler{"qwee", 5}, Handler{"qweee", 6}},
+		},
+	}
```

Comment:
In general it's preferable to use the name of the fields in literal structs declarations.
Why? If for whatever reason the order of the fields was modified in the struct definition : 
 - at best you'd have a compile error, for example the first field was an int and is now a slice
 - at worst the code still compiles because types are respected but now you're not assigning the wrong values to the fields!

Example:

```go
type BankTransaction struct {
    Cents int
    Dollars int 
}
```

to 

```go
type BankTransaction struct {
    Dollars int 
    Cents int
}
```
:D